src/wp-admin/options-writing.php

```diff
@@ -163,11 +165,21 @@
 <th scope="row"><label for="mailserver_login"><?php _e( 'Login Name' ); ?></label></th>
 <td><input name="mailserver_login" type="text" id="mailserver_login" value="<?php form_option( 'mailserver_login' ); ?>" class="regular-text ltr" /></td>
 </tr>
-<tr>
-<th scope="row"><label for="mailserver_pass"><?php _e( 'Password' ); ?></label></th>
-<td>
-<input name="mailserver_pass" type="text" id="mailserver_pass" value="<?php form_option( 'mailserver_pass' ); ?>" class="regular-text ltr" />
-</td>
+<tr class="mailserver-pass-wrap">
+	<th scope="row">
+		<label for="mailserver_pass">
+			<?php _e( 'Password' ); ?>
+		</label>
+	</th>
+	<td>
+		<input class="hidden" value=" " /><!-- #24364 workaround -->
+		<span class="wp-pwd">
+			<input type="text" name="mailserver_pass" id="mailserver_pass" class="regular-text" autocomplete="off" data-reveal="1" data-pw="<?php echo esc_attr( get_option( 'mailserver_pass' ) ); ?>" />
```

Comment:
```suggestion
			<input type="text" name="mailserver_pass" id="mailserver_pass" class="regular-text ltr" autocomplete="off" data-reveal="1" data-pw="<?php echo esc_attr( get_option( 'mailserver_pass' ) ); ?>" />
```

Per @sabernhardt's comment on the ticket.